docs/en/remote-flow.rst

```diff
@@ -19,22 +19,20 @@ Once the Wallet Instance establishes the trust with the Relying Party and evalua
 
 .. figure:: ../../images/cross_device_auth_seq_diagram.svg
     :align: center
-    :target: https://www.plantuml.com/plantuml/svg/XLNTRkCs4xtdKnpuk_YI84cpspi3AJQRfflijergUpSt1W9Q6jjiQ5BbZvoqwBjtA5goN2GeO1WGEMVcdFaPVl11kX0tMiWOh8cR1JaXn5KPhCFegoStWlI8GYpiSpxulyUFrAYI_0Z_-rcjrk6ZZYHgzgDALKiJC7gGMbF0dM7tfgJMn_RG9BAhCxaY4t84ASXKu2Y7PXFIBygiKF3XIslPa10HPpymP8m7Mc1ABj61aOrQoQ2i4L4cWuR6cD0VaNn0uMoWCb359zv6LxANoqiaGRYAOKs6OxPW71MEF7_dhs8jFHtZybB8CgX61aeC6ke2N3jAPn8d9vuXT7A2Hkf8SxEmaHK5YxrKbDO-gLVWBjDyZ4x1TGGRwcJ-Dhcs5U5-WjlPh609bNAi1aVVNxm_nbQ5DKgjEC2Zgaw7paxHaX8L3TcBlPQj42jdOyr4hEyg2slqoow9SOpfscx1AUferAmbbO1ljsFSrsJQstyL7RAkJw5jAJNaGOR55XawBQLXfqXU-yERv-2OtjpqYkPce6XkfAcoghbPEhj9iNidP2hHUw9K2BRp22xwaFEKh7lFsPJhqENCj_TUhGnLtHdiulUKx3xOi5WSiBRAXBPOEUohQ4wv3b3B8VXnGs5jAoWFJpjwnVrq9WG0zDHgPnjLaUyIp2XPoI0HxDjqaeU-omQP61GgK6bKgaufQxbGMWL5lWTcBR7cI5IzfUcDT1yCBNxf-AR2rLQhakFqg6_arFAb3I13T0Ix7DIWdz_x7B57IQqIi-wUBRyLH_uu6yJzt2-jGsbDckTzqyTzzrvRirHkGzLmH6BV5sqzsWLXdVepRuG9-JfvnDpacZNrOx1FPDxO43Uxl2HPMQwI2qquM3FPGG_g1GuBTXK2-Mn_pi7pvz2Wo9tYc9ZoI7kqo-PFnCSjak-u3gyEB6EC3N3KC9dgVC0cXqhBgafZB3Nkl8_aC4k3bLtkANz-c67QrNFWXUcLv7qUmTwL48wZ4cn3ZituJRdxT3lxDBcv-LMyZXWBNA-fVCAjrImBN-W57vMfkELQ8enYsVmhhFSCPzKCEKFqRD7wgkD2gn9F3CnRMgMyyCrGOh1eskRCXIjoZSw5m6nDfjjJxMMbVAcjfmeo1ASMCP0VSuaoppJsdEf-M1N7G9u8Hv2DL3erH9gcDXLYaBElTTS5-QtQwS5HVacw0Jt6O06bD-AxT-VlyFxY0lTVcCPV8vUVZ7PBZCPOXr1xHATp1xjTIp-MjYByFPo5xU7RqJqNKxcXWzSY3vvanZxJ7KPCqlxheMceV-S-JyUxp1q-77djuoW9x7PH5f2pSPoJa7bSlGPTgOPCXhHEabv_YiXsMeEUwxOsri77CdNUCkgM-mV-Ynlz3m00
-
+    :target: https://www.plantuml.com/plantuml/uml/ZLLRRnit4ttdhpZmz_3PXkCcwPi05t5SDvUqjMefoKCI61Ht9B6co2qlgxX5_tjdIN6zAbIqW21WoRcSSsQ6yo94wMFQ625JT3Pj5kI549SgEe-Zzu7y4MH04cBXlfQ3_mTyJvrM58xmln_rQXrVcaIYvJXQwLRQYAEFqXYAU6Cvl5MKOsptJHA7UY9NP6F980NP1fnbt-oMp8EsAqpvy25RTYH4mfFls6M2ZxJ0Z7mF1rPsUYMZTbOasHjqDC4j_POeH4ozYwGYDNmINFAAernSe2U4onJNP3bdTiRnr5FWx_8rNSj09xYu4595tSb8FOAEQe8hJ_sPnEbfO1SwFKKdJQBFTk4ICueEZrIXdBlN1znzWNcNl8Ql03kgD7-vlN8DuUOMh-VpALDej6SsEFxswVCJR7LSVxRAQCvmfvkpSRGpP5Hd-5GFDnJNhkdllk3Ju7GNr9esz4KqDEOf7bblXLXvbrg6x4Bj0JXh7dPofmbcQxS5Yra1jyowBnRVRYUuQSdcsx-r1sJRdi4u9GtaNYt45iLrfgJbeEYZ3eVNNwDdU3baAxraPkG3bjvjybcr9zk8mOdwwZ2VWEXJY5cyR3r8SKWL8Ks4NewLtGa97H1e5sTwCviB0DZM-hRUTRgXK3cpasnab8YEi-uNdd6sJjkO8zGM6I4UQnT9MTqBfcGh5zheg2a8Ce7-0ynNrtCu3-iYsjBhOeIvUfTyGQyiDVKCRl4hgLWrFeo_2DvHhVl14E9GIVQc7JzLhQHjq3gZ5NCSl3wGryk775p6v0bkEGPS1_-cP1kyfD4Sa-ezMtwO7eUJOf0cWz_IUIXdipZ33AJ7PhvoTgdzXfhnn2HHHSoja5yMv_NlkPavT6ZxYFXiInswzYwwUxB_yBHSxtJfQHMwMYuCOUzzAW6gquZWg5akOwHGxCyvUVaS6qnBbte6faC_Ix6g-GFnSVRaEwx3mVdL8VUIfux1DESZDBjKLvwQVggbmPiG4ddC--7RXHmWl_qunwpMSnI5uIaAeOpf44e8pra9hYFFPhoCthnwFNzrUttzipZiCH6uMfEwnsjdc1RUqGDUQjjmosKWxCMq-JTr8R0xisQN9qY1BSepanP6AzopOtdUjStd8uBHeSkjMSesBHKw5mBpDTczHwlHGHaNg86CqUj5YXGcK1cYNe9L7cPCSmAKOV04sKbj-bxeHosh84PolMzh0vJzjgE2ux9zaF9zSuBHZFPxvPjdvy_m_EbJtBpZolnnvCkQoSDzDC4wnZOWULEQr--w-KwvXl3dU0os-rkaBewPtD3UtjZM_SOm2zDHnfxQlwsSZeRT-7OssHLtVuMK_OVZdc1zWWg1eiP94Q7Wk3pakj8TsUAIW_HymnaIsna3-jhcsZYFa5JVTCjaClON_9SsvYy0
```

Comment:
```suggestion
    :target: https://www.plantuml.com/plantuml/uml/ZLLRRnit4ttdhpZGz_3PXkCcwPi05t5SDvUqjMefoKCI21Ht9B6co2qlgxX5_tjdIUo2SWIqXc5Or5pEd3apyo94wMFQ6I5JT3RjLkI545SgEe_9-q1-0XcGX5Yvh-NX_m4_KgSLXGd-zxFUREDBaqYq74ShtKeRCVaeZQ68DvR3MqKvnlxG976e9t93DfOCKf1jm5aEpUx8F6YxmOmV7x9bHq985NKd8p6mX2S_iFSK7sc5EVaU6Qpiz4P6xMnbAaL3jQFiFJkQuex-I1GYPjv5Kf4QVWakUSMPBY_vcGB3pE4msyf0kBmSu_PuEXf2NNlgtoJEpW9xqDaPEidqVBOhbxY-w30M7g6SkzT7t7q1j4nUmmzGEXeg_UtksZM4spkyNYnc3BRHdd7ZvFkdpq-nrt7xsIgZESDID-Trj7DaLEVuL3qkTnIthlc2JwFJCgWjxUXBnLZcdNystgMMWRemw03EceT29z8KywtR8svfWDbohGVCoqVjOxVz5BSjsUpsI3kGRNiEuvGqa8aDB8kZjBOEEiPadUrHGiwLj8m2FOpnyUPhz6oRKx7I91TFFc3jtIw_3VPa6p4wKTz43XuGVGenSvojHyQEaIUaYT1hVCirT42W4WUps_b20C1-a47hNQSR2WCNx2GR6IMYorbhVJ3ErcaRSuIwMG0mAgkgeE8uahAw9rR9LgwqqL5J46G2_GVOhsuMS1xMHRHbGaHLm-1TyWMziTGCRt2htqd5aVFZzUG6T3JeusY8n_eyqh231O9V1YQbRT0wenLNF7fz8A_N3pcLZCaJtBg2F-w9sv9xz996DzNOikM7sKauBGPfElgjdOToDbDrR0IkdLrhlwFcKDa3fLaek4hS2Q__BEk85rNRHyHdMxhIjSlexicEazvMssiPqcgdG-1_Z-ylt1JUA9UTwT1oPUXm1-O4erD5G2MjgyZa5_iB1bS-rWQped7Fm7L0l_C7jdyYViwU_1ttwUXMq24hprF7O9gB2ar9rQ_IXX5hQV0cX4GUYtvqIyWHCjv-N60-zBkAml0KXQ9UDOOImdDMWkl8indVlpUlNYzUtTtS_2gEUqm5hhUaxl7QcOQTtj43NchRi2Gp4FRYilbtvaAWt1fsUO8KsA8igRHYcUAU7SiDqNbFrWL6XovBjxHjLZfq8GGMMxAWQkKrEYZZ0ZMGCNgzIapM52f3r6imo-D4Qaw0CYoU83kf3MiXRyRCWgZeUj-E0dWVo2EKKd1IJXnI_huvHJNH_V7wxjd5Cpn_-XItxxWpVqrykMLv5KocsCRO1YHlYhWz5RKlbMi8Vul7urzkqrpkwE0qxOV3oT0on3fB7MClwYl5TVxRkUjelCPlvzFIwKTNgimXJlfgn-sLgodKd7upYGKihew5sUYER77F0NiYmZaIsnd3ZMvpqvn7IAfNupgPmzjxI5ckyGN_IJlp3m00
```

Reply:
I made the following chances to the flow:
1. remove the term “ User Authentication”
2. request_method_uri —> request_uri_method
3. add all the parameters in step 20
4. Extract —> extract
5. Validate Requested VP(s) —> Validate requested VP(s)
6. adjust activate/deactivate